source/services/piri_service.txt

```diff
@@ -0,0 +1,51 @@
+DataONE PIRI Service
+====================
+
+Overview
+--------
+
+The DataONE PIRI (Persistent Internationalized Resource Identifier) Service provides canonical and resolveable IRIs (Internationalalized Resource Identifier) for top-level DataONE resources (e.g., datasets, people) in both web and linked open data contexts.
+The problem this solves is that DataONE resources can be resolved at a variety of URLs which contain application-, deployment-, and API (Application Programming Interface)-version-specific details.
+An IRI space that is agnostic to these details allows resources to be identified and resolved in a stable fashion.
+
+Take, for example, the following IRI, which can be used to resolve the location of the Object with the PID_. (Persistent Identifier) 'mydataset':
+
+.. _PID: https://dataone-architecture-documentation.readthedocs.io/en/latest/design/PIDs.html
+
+`https://cn.dataone.org/cn/v2/resolve/mydataset`
+
+Under the PIRI Service, the above resource has the following PIRI:
+
+`https://dataone.org/datasets/mydataset`
+
+This is an improvement because the first IRI leaks application-, deployment-, and API-version-specific information into the IRI.
+All of which may change as some point in the future.
+With the DataONE PIRI Service (and the IRI space it defines), applications become free to be addressed at different DNS (Domain Name Service) names and APIs are free to change and/or have their versions incremented.
+All while references to resources within the network remain resolveable.
+
+Specification
+-------------
+
+The DataONE PIRI Service defines an IRI space with the following pattern:
+
+`https://dataone.org/{type}/{id}`
+
+Where `{type}` is one of:
+
+- `datasets`
+- `people` (not implemented)
+
+And `{id}` is an appropriate identifier for resource such as a PID or SID (Series Identifier) for the 'datasets' type.
+Note that all identifiers should be escaped according to RFC3986_.
+See `Identifiers in DataONE`_ for more information.
+
+.. _RFC3986: https://tools.ietf.org/html/rfc3986
+.. _Identifiers in DataONE: https://dataone-architecture-documentation.readthedocs.io/en/latest/design/PIDs.html
```

Comment:
Thanks for adding this. The absolute URI you used might break, so could you switch to using an inline link to the documentation, which will set up a relative link? The syntax is ":doc:`design/PIDs`", and means that links will work even when the docs are built and deployed on different sites. There might be a few of these in your text.

Reply:
Good catch.